tests/subsys/power/power_mgmt/src/dummy_driver.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <sys/printk.h>
+#include <zephyr/types.h>
+#include "dummy_driver.h"
+
+#ifdef CONFIG_DEVICE_POWER_MANAGEMENT
+uint32_t device_power_state;
+
+static int dummy_open(struct device *dev)
+{
+	int ret;
```

Comment:
here and throughout, please initialize `ret`. It is used for decisions and initializing helps keep things starting where intended rather than starting from potentially different/unexpected value.